src/test/java/elysia/storage/StorageTest.java

```diff
@@ -0,0 +1,103 @@
+package elysia.storage;
+
+import elysia.command.Deadline;
+import elysia.command.Event;
+import elysia.command.Task;
+import elysia.command.ToDos;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
static import should go before normal import and also careful of wildcard import by IDE!